### PR TITLE
Dataset Analysis - `contributions_quantification.ipynb`

### LINKED ISSUE #56: dataset notebook to show of a data provider's data
grab datasets from list of ids:

```python 
datasets_of_interest = [
    "UUID_1",
    "UUID_2",
]
```

* [ ] spatial distribution
* [ ] temporal distribution
* [ ] taxanomic distribution
* [ ] what MoFs are included?
* what makes this dataset unique?

### PR DESCRIPTION
**Pull request by ayushanand18**
## Overview
This notebook analyses datasets contributed by individuals and performs,
+ spatial distribution analysis
+ temporal distribution analysis
+ taxonomic distribution analysis
+ and conclusions on how these datasets are unique

## Notes for reviewers
To circumvent merge conflicts in the previous PR #62, I had to switch to a new branch and a new PR.

Tries to resolve #56

### CODE REVIEW

**Comment by ayushanand18** [comment]
Update: I have added the `geometry` into the analysis and also update the charts accordingly. Most charts would be visible but some including the interactive pie chart and gantt-style chart won't be visible on GitHub to their dynamic nature, they work fine when opening via Google Colab (through the badge on top).

File: notebooks/contributions_quantification.ipynb [hunk 1]
```diff
@@ -6,7 +6,7 @@
       "name": "contributions_quantification.ipynb",
```
Comment (7yl4r) [inline notebooks/contributions_quantification.ipynb:6]: <p>Why doesn't this data match the printout above? </p><p> </p><blockquote>Michelle Vukovich       253939 </blockquote><blockquote>Tim Cole                 21412 </blockquote><blockquote>Beth Josephson           10948 </blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/iobis/pyobis/pull/67/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Comment (ayushanand18) [reply notebooks/contributions_quantification.ipynb:6]: I don't know why they aren't matching. I queried through dataset endpoint and it resulted into those printed values, but with the same parameters in the search query when I fetched occurrence records, the number was different. The total length of the occurrence records fetched was 104621 which is way smaller than it should have been. When I run a similar visualization on the data from dataset endpoint I get a graph like: 

image

Comment (7yl4r) [inline notebooks/contributions_quantification.ipynb:6]: <pre class="ql-syntax" spellcheck="false">Line #1.    record_ids = {}
</pre><p>add comment here: # select the top data set from each of the top three authors</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/iobis/pyobis/pull/67/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (ayushanand18) [reply notebooks/contributions_quantification.ipynb:6]: Update: Thank you for pointing this out, I have updated the notebook.

**Pull request merged**